MongoDbChecker/MongoDbChecker.go

```diff
@@ -35,7 +35,6 @@ func (m *MongoDbConnection) Validate() (bool, error) {
 
 func (m *MongoDbConnection) Check() (bool, bool, error) {
 	client, err := mongo.NewClient(options.Client().ApplyURI(m.buildConnectionString()))
-
```

Comment:
What is this whitespace?

Reply:
it's removed. but probably some special magic was behind this whitespace.